### ISSUE TITLE
Maintenance 

### ISSUE THREAD

**Issue by Cigar1** [tanjeffreyz/auto-maple-resources#16]
Does it need to be updated after Maintenance ? 

just to be safe

**Comment by LiliaTheSuccubus** [comment]
No, because it doesn't have anything to avoid detection. However since it doesn't inject into Maple or interact with memory, I don't think Nexon cares, but I could be wrong.
I'm still new to this, but I doubt I'll have any issues running it or my other Python projects alongside Maple.
